### PR TITLE
Grav star plus doc changes

### PR DESCRIPTION
**Pull request by fvnkhead**
* -33% pull time for grav star
  * README updates
  * `diffs.sh` and `changes.sh` updates to work with both keyvalues and Squirrel

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,18 +11,19 @@ Goals
   3. Minimal changes
   4. Avoiding _scope creep_ which causes unnecessary changes to accumulate
 
-Constaints
```
Comment (fvnkhead) [inline README.md:14]: lol

File: mod/scripts/vscripts/retouched.nut [hunk 2]
```diff
@@ -66,9 +67,17 @@ global array< array<string> > RETOUCHED_CHANGELIST = [
...
+    [
+        "Grav",
+        "Pull time: 2.0s -> 1.33s (-33%)"
+    ],
     [
         "Melee",
         "Damage: 100 -> 70"
+    ],
+    [
+        "Github",
```
Comment (fvnkhead) [inline mod/scripts/vscripts/retouched.nut:79]: This adds `!github` command if someone wants to find the repo

File: mod/scripts/vscripts/weapons/mp_weapon_grenade_gravity.nut [hunk 3]
```diff
@@ -0,0 +1,382 @@
+global function OnProjectileCollision_weapon_grenade_gravity
+global function MpWeaponGrenadeGravity_Init
+
+const float MAX_WAIT_TIME = 6.0
+const float POP_DELAY = 0.8
+const float PULL_DELAY = 1.33 // CHANGE/NERF: 2.0 -> 1.33 (-33% pull duration)
```
Comment (fvnkhead) [inline mod/scripts/vscripts/weapons/mp_weapon_grenade_gravity.nut:6]: The changed line

File: README.md [hunk 4]
```diff
@@ -11,18 +11,19 @@ Goals
...
+Constraints
 --------------------------------------------------------------------------------
 
 For now:
 
-  1. Damage changes only, no messing with other variables like falloffs or headshot multipliers
-  2. SMG, R201/101, Flatline, EVA-8 and Spitfire changes only
-  3. No tactical and ordnance changes (yet)
+  1. Changing 1 value only per item (eg. damage for guns, pull time for gravity star, etc.)
```
Comment (fvnkhead) [inline README.md:19]: I have altered the constraints, pray I do not alter them any further

Comment (SinB-XoXo) [reply README.md:19]: i agree to this completely

File: README.md [hunk 5]
```diff
@@ -11,18 +11,19 @@ Goals
...
 
-Constaints
+Constraints
 --------------------------------------------------------------------------------
 
 For now:
 
-  1. Damage changes only, no messing with other variables like falloffs or headshot multipliers
-  2. SMG, R201/101, Flatline, EVA-8 and Spitfire changes only
-  3. No tactical and ordnance changes (yet)
```
Comment (SinB-XoXo) [inline README.md:21]: Ordnance change already made for testing

**Review by SinB-XoXo — APPROVED** [review]

File: README.md (hunk 5 above)
Comment (fvnkhead) [reply README.md:21]: Removed line

**Pull request merged**